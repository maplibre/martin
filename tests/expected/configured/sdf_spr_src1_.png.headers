content-length: 1699
content-type: image/png
etag: W/"6a3-NdhmitklvJKSk_7phaxquw=="
vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
