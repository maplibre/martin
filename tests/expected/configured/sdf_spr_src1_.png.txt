tests/output/configured/sdf_spr_src1_.png: PNG image data, 120 x 46, 8-bit gray+alpha, non-interlaced
